build_modules/lib/src/platform.dart

```diff
@@ -120,3 +63,22 @@ class DartPlatform {
   @override
   bool operator ==(other) => other is DartPlatform && other.name == name;
 }
+
+class DartPlatformAlreadyRegisteredException implements Exception {
```

Comment:
These are really long - I wonder if it would be just as readable with without the trailing `Exception` in the name?